sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj

```
@@ -6,12 +6,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Support</PackageId>
     <Description>Azure Support SDK allows creation and management of Azure support tickets. Using the operations for support case management, you can (1) Open a technical, billing, subscription management, or quota increase support ticket, (2) Get list and detailed information about support tickets, (3) Update severity, contact information of a support ticket (4) Add a new communication and get list of communications for a support ticket.</Description>
-    <Version>1.0.0-preview.1</Version>
+    <Version>1.0.0.0</Version>
```

```suggestion
    <Version>1.0.0</Version>
```
Please use 3-digit for project versioning